src/dotnet/commands/dotnet-add/Program.cs

```diff
@@ -0,0 +1,69 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using Microsoft.DotNet.Cli.CommandLine;
+using Microsoft.DotNet.Cli.Utils;
+using System;
+using System.IO;
+using System.Linq;
+using Microsoft.Build.Construction;
+using Microsoft.DotNet.ProjectJsonMigration;
+using NuGet.Frameworks;
+using Microsoft.DotNet.Tools.Add.ProjectToProjectReference;
+
+namespace Microsoft.DotNet.Tools.Add
+{
+    public class AddCommand
+    {
+        public const string CommandName = "dotnet-add";
+        public const string UsageText = @"Usage: dotnet add [command] [arguments]
+
+Arguments:
```

Comment:
I didn't use the parser for the same reason we didn't in dotnet command - the parser doesn't work here and I didn't wanted to go sideways and invent my own parser. I was trying to make the errors the same but seems like I need to work on that a little more. I thought we will unify that when adding tab completion.